svn/common.py

```diff
@@ -54,8 +54,9 @@ def run_command(self, subcommand, args, success_code=0,
                              stderr=subprocess.STDOUT,
                              env={'LANG': 'en_US.UTF-8'})
 
-        stdout = p.stdout.read()
         r = p.wait()
+        stdout = p.stdout.read()
```

Comment:
Most likely you are right, I looked at ``communicate()`` method and it behaves similarly, so I returned read from stdout where it was.